pages/index.vue

```diff
@@ -1,27 +1,66 @@
 <template>
   <div>
-    <Homepage />
+    <Homepage :showInstallMessage="showInstallMessage" />
   </div>
 </template>
 
 <script>
+import metadata from '../static/content/metadata.json'
 import Homepage from '../components/Homepage.vue'
 
 export default {
   components: {
     Homepage
   },
+  data() {
+    return {
+      metadata: metadata,
+      showInstallMessage: false
+    }
+  },
   created() {
-    if(process.client) {
+    if (process.client) {
+      // eslint-disable-next-line
       if (window.netlifyIdentity) {
-        window.netlifyIdentity.on("init", user => {
+        // eslint-disable-next-line
+        window.netlifyIdentity.on('init', user => {
           if (!user) {
-            window.netlifyIdentity.on("login", () => {
-              document.location.href = "/admin/";
-            });
+            // eslint-disable-next-line
+            window.netlifyIdentity.on('login', () => {
+              // eslint-disable-next-line
+              document.location.href = '/admin/'
+            })
           }
-        });
+        })
+      }
+
+       // Detects if device is on iOS 
+        const isIos = () => {
+        // eslint-disable-next-line
+        const userAgent = window.navigator.userAgent.toLowerCase()
+           return /iphone|ipad|ipod/.test( userAgent )
+        }
+        console.log(isIos())
```

Comment:
Try to have your own console log wrapper so we can easily turn it on / off in different environment

Reply:
Ok, noted. I will try to use a wrapper in future. I have removed this console.log statement.